pkg/params/run.go

```diff
@@ -3,6 +3,7 @@ package params
 import (
 	"context"
 	"fmt"
+	"log"
```

Comment:
is it possible to use existing suggared logger?

Reply:
Yes we can
Updated